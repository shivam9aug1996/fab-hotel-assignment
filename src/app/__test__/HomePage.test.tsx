import HomePage from "@/app/page";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

describe("HomePage", () => {
  it("renders homepage", () => {
    const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });
});
