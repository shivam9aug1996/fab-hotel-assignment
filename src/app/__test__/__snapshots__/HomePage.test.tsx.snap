// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders homepage 1`] = `
<div>
  <div
    class="relative p-16 rounded-lg bg-slate-300 text-center px-32"
  >
    <h1
      class="text-3xl font-bold text-black font-serif"
    >
      Welcome to
    </h1>
    <span
      class="inline-flex items-center justify-center text-gray-800 bg-yellow-400 rounded-full w-10 h-10 mr-1"
    >
      fab
    </span>
    HOTELS
  </div>
</div>
`;
